README.md

```diff
@@ -120,6 +120,29 @@ and tactics specialized for monadic programs (see
 
 A tutorial for the Lean backend is available [here](./tests/lean/Tutorial.lean).
 
+## Quick start for Nix users
+
+Assuming Nix is installed, with a support for Flakes (`*`):
+
+```console
+$ nix-shell -p rustup
+$ nix run github:aeneasverif/aeneas#charon -L
+$ # The default package for Aeneas is Aeneas.
+$ nix run github:aeneasverif/aeneas -L -- -backend your_preferred_backend your_llbc.file
+```
+
+To regenerate the extraction, just run step 2 and step 3 again.
+
+It is critical not to use the Charon repository and rely on the locking
+performed in this repository to avoid incompatible LLBC files.
```

Comment:
Explain that sometimes it may happen that Charon and Aeneas are desynchronized: for instance some changes to the LLBC in the side of Charon haven't been merged to Aeneas yet. We try to avoid this as much as possible, but in you're using Nix, there is a simple way of avoiding the problem, etc.

Reply:
See also https://github.com/AeneasVerif/charon/issues/35, which I've got a plan for if I ever find the time

Reply:
Yes, I agree this is pertinent. I just want the comments to be modified to take into account my remarks above, and then I approve the PR. :)

Reply:
Addressed!